examples/headers/update.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Version of libbpf to fetch headers from
+LIBBPF_VERSION=0.4.0
+
+# The headers we want
+headers=(
+    bpf_helper_defs.h
+    bpf_helpers.h
+)
+
+# Fetch libbpf release and store in /tmp
+curl -sL "https://github.com/libbpf/libbpf/archive/refs/tags/v${LIBBPF_VERSION}.tar.gz" -o /tmp/libbpf.tar.gz
```

Comment:
Maybe write to `$(mktemp)` and add a `trap EXIT` to remove the file?

Reply:
I think I've circumvented the need for a temp file in the latest push.